src/idl_gen_swift.cpp

```diff
@@ -0,0 +1,719 @@
+#include <unordered_set>
+
+#include "flatbuffers/code_generators.h"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/idl.h"
+#include "flatbuffers/util.h"
+
+namespace flatbuffers {
+
+namespace swift {
+
+inline std::string GenIndirect(std::string reading) {
```

Comment:
V813 Decreased performance. The 'reading' argument should probably be rendered as a constant reference. idl_gen_swift.cpp 12